tests/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+set(UNITTESTS_SRC
+        unittests/unittests.cpp
+        unittests/mallocmacro.cpp
+        common/simplethread.cpp
+        common/systemtime.cpp
```

Comment:
As I understand, it doesnt build c_api files as a separated library, so the current approach will have the error like:
```c
/usr/bin/ld: unittests.cpp:(.text._ZN10moodycamel20ConcurrentQueueTests13c_api_enqueueEv[_ZN10moodycamel20ConcurrentQueueTests13c_api_enqueueEv]+0x56): undefined reference to `moodycamel_cq_destroy'
```
Because of that, it's important to add c_api files to `UNITTESTS_SRC`, isn't it? (or add a target to build c_api library)
```c
set(UNITTESTS_SRC
    unittests/unittests.cpp
    unittests/mallocmacro.cpp
    common/simplethread.cpp
    common/systemtime.cpp
    ../c_api/concurrentqueue.cpp
    ../c_api/blockingconcurrentqueue.cpp
)
```

Reply:
I don't think the c_api contribution existed back then.

Reply:
Oh, okay, in this case it is a necessary to add c_api support to CMake build system.